.travis.yml

```diff
@@ -25,6 +25,10 @@ install:
 - pip install -r requirements-dev.txt
 - wget -q ${ES_DOWNLOAD_URL}
 - tar -xzf elasticsearch-${ES_VERSION}.tar.gz
+- wget -q http://us-east-1.ec2.archive.ubuntu.com/ubuntu/pool/main/m/make-dfsg/make_4.1-6_amd64.deb
+- mkdir make4
+- dpkg -x make*.deb make4
+- export PATH=$(pwd)/make4/usr/bin:$PATH
 script:
 - set -eo pipefail
 - make test
```

Comment:
I think you should consider making this -j -O.

Reply:
That's coming in a separate PR :)